src/components/TopicMenu/TopicMenu.js

```diff
@@ -207,12 +210,14 @@ class TopicMenu extends PureComponent {
               </div>
               {t(topic.name)}
             </div>
-            <div
-              className={`wkp-layer-toggler ${collapsed ? 'collapsed' : ''}`}
-              style={{
-                display: topic.key === activeTopic.key ? 'block' : 'none',
-              }}
-            />
+            {isMenuVisibleLayers ? (
+              <div
+                className={`wkp-layer-toggler ${collapsed ? 'collapsed' : ''}`}
+                style={{
+                  display: topic.key === activeTopic.key ? 'block' : 'none',
+                }}
+              />
+            ) : null}
```

Comment:
As I said in a previous comment when it's a boolean you can do : 
```suggestion
            {isMenuVisibleLayers && (
              <div
                className={`wkp-layer-toggler ${collapsed ? 'collapsed' : ''}`}
                style={{
                  display: topic.key === activeTopic.key ? 'block' : 'none',
                }}
              />
            )}
```